examples/walkthrough/README.md

```diff
@@ -121,6 +121,46 @@ Note that we have also added a volume here.  In this case, the volume is mounted
 
 Finally, we have also introduced an environment variable to the ```git-monitor``` container, which allows us to parameterize that container with the particular git repository that we want to track.
 
+### Containers and commands
```

Comment:
This detailed explanation belongs in a doc, not in an example.

Please move it to:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/containers.md